wordpress-mobile/WordPress-iOS
# ULS: Show nav bar with updated Help button

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/315
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/327

This updates Auth to show the new unified nav bar, i.e. the `Help` button is now blue. Rejoice.

To test:

---
- Enable the `unifiedSiteAddress` Feature Flag.
- Go to Log In > Site Address.
- On both views in this flow, verify the unified nav bar is displayed with the blue `Help` button.

| Site Address | Site Credentials |
|--------|-------|
| ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-10 at 12 18 07](https://user-images.githubusercontent.com/1816888/87186624-0ce7c980-c2a9-11ea-8cfe-81008ab69608.png) | ![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-10 at 12 18 18](https://user-images.githubusercontent.com/1816888/87186631-1113e700-c2a9-11ea-8d60-8b0e3e421ce8.png) |

---
- Go to any other auth view.
- Verify the old nav bar is displayed.

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-10 at 12 26 15](https://user-images.githubusercontent.com/1816888/87186714-33a60000-c2a9-11ea-9378-0f03c47c1246.png)

---
PR submission checklist:
